contracts/exchange/api/BaseContract.sol

```diff
@@ -337,4 +349,45 @@ contract BaseContract is AccessControlUpgradeable, ReentrancyGuardUpgradeable {
 
     return pos;
   }
+
+  function _deployDepositProxy(address account) internal returns (address) {
```

Comment:
Assume this function will be read by auditors, let's add comments explaining this function here

Reply:
Added comment